src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoApp.scala

```diff
@@ -66,21 +68,21 @@ trait EasyAuthInfoApp extends AutoCloseable with DebugEnhancedLogging with Appli
   }
 
   private def fromBagStore(bagId: UUID, path: Path): Try[Option[CachedAuthInfo]] = {
-    logger.info(s"[$bagId] item for path $path not found in autCache, trying to retrieve item from bagStore")
+    logger.info(s"[$bagId] item for path ${path.escapePath} not found in authCache, trying to retrieve item from bagStore")
     bagStore
       .loadFilesXML(bagId)
       .map(getFileNode(_, path))
       .flatMap {
         case None => Success(None) // TODO cache repeatedly requested but not found bags/files?
         case Some(filesXmlItem) =>
-          collectInfo(bagId, path, filesXmlItem).map { fileItem =>
+          collectInfo(bagId, path.escapePath, filesXmlItem).map { fileItem =>
```

Comment:
can you move the `.escapePath` to the `FileItem` constructor below. That way we keep the `path` variable of type `Path` and do the escaping as late as possible.

Reply:
or even better... move the `.escapePath` into the `FileItem.solrLiterals`. That way we even keep the value in `FileItem` a `Path`.

Reply:
👍 I fully agree with this comment. I'll do the change myself, as this issue is taking too long already.